part2/stages/Configure-partitions

```diff
@@ -398,23 +401,6 @@ partition_mode_is_unique()
             fi
             ;;
 
-        erase-entire-disk)
-            # Check if equivalent to "overwrite".
-            if [ "${XC_PARTITION}" -a \
-                 -z "${OEM_PARTITIONS}" -a \
-                 -z "${OTHER_PARTITIONS}" -a \
-                 "${HAVE_FREE_SPACE}" != "true" ] ; then
-                return 1
-            fi
-
-            # Check if equivalent to "use-free-space".
```

Comment:
@tklengyel do you remember why this changed to a no-op?  When booting the installer in legacy mode, erase-entire-disk is no longer considered identical to use-free-space causing a prompt for partitioning.
https://openxt.atlassian.net/browse/OXT-1349

Reply:
I remember erase-entire-disk was not showing up in the installer as an option so we tried a bunch of tweaks and ultimately had to specify the partition-mode in the answer-file specific to UEFI to get it to work. We never got it to show up in the interactive installer as an option so this might just have been one of the tweaks we never reverted after we tried it.